deploy/generic-k8s/ecommerce-ads-versions/advertisements_v1.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    tags.datadoghq.com/env: development
+    tags.datadoghq.com/service: advertisements
+    app: ecommerce
+  name: advertisements
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      tags.datadoghq.com/env: development
+      tags.datadoghq.com/service: advertisements
+      app: ecommerce
+  strategy: {}
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        tags.datadoghq.com/env: development
+        tags.datadoghq.com/service: advertisements
+        tags.datadoghq.com/version: "1.0" 
+        app: ecommerce
+    spec:
+      containers:
+      - image: public.ecr.aws/x2b9z2t7/ddtraining/ads_versions_v1:latest
```

Comment:
Since you're referencing the ECR repo here with these new images, why do we need to cross push to Dockerhub as well?